eng/tools/versioning/VersionUtils.js

```diff
@@ -92,6 +93,21 @@ function buildSemverRegex(prefix) {
   return new RegExp(`(${prefix}.*?)(${semverRegex.toString()})`, "g");
 }
 
+function updateChangelog(targetPackagePath, newVersion, unreleased, replaceVersion) {
+  const changelogLocation = path.join(targetPackagePath, "CHANGELOG.md");
+  child = spawn("powershell.exe", ["eng/common/Update-Change-Log.ps1", newVersion, changelogLocation, unreleased, replaceVersion]);
```

Comment:
I think you want to call powershell core so this should be `pwsh` otherwise it will not work cross-platform.

Reply:
done